### PR TITLE
Validating drawer header exists before trying to modify its padding size

### PR DESCRIPTION
**Pull request by War777**
This fix must ensure that **_onHeaderUpdate** function validates the drawer header exists before modifiying its padding.

[azure devops ticket here](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/28059/)

### CODE REVIEW

File: src/templates/drawerDeprecated.js [hunk 1]
```diff
@@ -147,13 +147,12 @@ class DrawerDeprecatedHeader extends Component {
 
     _onHeaderUpdate(node) {
         const headerEl = ReactDOM.findDOMNode(this);
-
         if (headerEl) {
             const paddingTop = headerEl.offsetHeight;
-
-            if (paddingTop !== this._drawerContainerInnerPaddingTop) {
+            let drawerInnerContainer = headerEl.closest('.drawer-container-inner');
```
Comment (sircris) [inline src/templates/drawerDeprecated.js:152]: I suppose that `drawer-container-inner` is a standard class for the container? Is this a safe assumption?

**Comment by sircris** [comment]
Deferring approval and final review to @morethanfire

File: src/templates/drawerDeprecated.js (hunk 1 above)
Comment (War777) [reply src/templates/drawerDeprecated.js:152]: Yes, it is an standard, the container, element with class it's created inside the render method, I suggest to move the class name to a constant in a separate task.

Comment (morethanfire) [reply src/templates/drawerDeprecated.js:152]: Yes, that is a safe assumption. It looks that `drawerInnerContainer` is not being redefined and should be a `const`.

Comment (War777) [reply src/templates/drawerDeprecated.js]: Done! Commit pushed :) thanks.

**Pull request merged**